test/Common/DurableClientBaseTests.cs

```diff
@@ -212,6 +228,30 @@ public void DurableClient_ExternalApp_CreateHttpManagementPayload_ThrowsExceptio
             Assert.ThrowsAny<InvalidOperationException>(() => durableOrchestrationClient.CreateHttpManagementPayload("testInstanceId"));
         }
 
+        /// <summary>
+        /// End to end test that ensures that DurableClientFactory is set up correctly
+        /// (i.e. the correct services are injected through dependency injection
+        /// and AzureStorageDurabilityProvider is created).
+        /// </summary>
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task DurableClient_AzureStorage_SuccessfulSetup()
+        {
+            string orchestratorName = nameof(TestOrchestrations.SayHelloInline);
+            using (ITestHost host = TestHelpers.GetJobHost(
```

Comment:
Just to make sure, does this path fail if `DurableTaskJobHostConfigurationExtensions` changes aren't present?

Reply:
Yes, this test fails with an `InvalidOperationException` saying it's unable to resolve a service.